src/main/java/com/labsynch/labseer/api/ApiParentLotController.java

```diff
@@ -126,34 +127,37 @@ public ResponseEntity<String> updateLotMetaArray(@RequestBody String json) {
 		}
 	}
 
-	@Transactional
 	@RequestMapping(value = "/reparentLot", method = RequestMethod.POST, headers = "Accept=application/json")
 	@ResponseBody
-	public ResponseEntity<String> reparentLot(@RequestBody String json) {
+	public ResponseEntity<String> reparentLot(@RequestBody String json, @RequestParam(value = "dryRun", required = false, defaultValue = "true") Boolean dryRun) {
 		HttpHeaders headers = new HttpHeaders();
 		headers.add("Content-Type", "application/json; charset=utf-8");
 		try {
 			ReparentLotDTO lotDTO = ReparentLotDTO.fromJsonToReparentLotDTO(json);
-			Lot lot = lotService.reparentLot(lotDTO.getLotCorpName(), lotDTO.getParentCorpName(),
-					lotDTO.getModifiedBy());
-			return new ResponseEntity<String>(lot.toJsonIncludeAliases(), headers, HttpStatus.OK);
+			ReparentLotResponseDTO reparentLotDTO = lotService.reparentLot(lotDTO.getLotCorpName(), lotDTO.getParentCorpName(),
+				lotDTO.getModifiedBy(), true, true, dryRun);
```

Comment:
The hard coded boolean are for updating lot inventory and containers which we always want to do with this route.  I imaged other scenarios where we wouldn't want to do this.  I.e. if there was an admin utility which was reparenting a lot and then replacing the original lot so it wanted to leave the assay and inventory data inlace.